_sass/_styles.scss

```diff
@@ -195,7 +196,11 @@ Does not work on round things. */
 
 #fall-competition-2015 {
   background-image: url(../assets/images/gallery-photos/fall-comp-2015-5.jpg);
-  background-position: 0 60%;
+  background-position: 0 70%;
+}
+
+#spring-competition-2015 {
```

Comment:
If you’re talking about the change from 60% to 70%, that was a change that Vivien and I made a while back. I guess we didn’t push the change, because it was such a minor change.

If you’re talking about the empty url for the #spring-competition-2015 background-image, it’s empty because I’m still kinda confused about what we’re doing for the gallery. Are we putting everything on Google Drive, or are we going to serve compressed images to the website? I guess for a short term solution, we could just compress all of the photos in Photoshop and then upload them to the repo.


> On Jan 8, 2016, at 8:11 PM, Christine Wang <notifications@github.com> wrote:
> 
> In _sass/_styles.scss <https://github.com/pioneers/website/pull/43#discussion_r49260065>:
> 
> > @@ -195,7 +196,11 @@ Does not work on round things. */
> >  
> >  #fall-competition-2015 {
> >    background-image: url(../assets/images/gallery-photos/fall-comp-2015-5.jpg);
> > -  background-position: 0 60%;
> > +  background-position: 0 70%;
> > +}
> > +
> > +#spring-competition-2015 {
> Looks like you didn't finish the styles for this part?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/pioneers/website/pull/43/files#r49260065>.